Src/FluentAssertions/Collections/GenericCollectionAssertions.cs

```diff
@@ -1845,7 +1845,7 @@ public AndConstraint<TAssertions> NotBeEmpty(string because = "", params object[
 
         using (var scope = new AssertionScope())
         {
-            Subject.Should().BeEquivalentTo(unexpected, config);
+            BeEquivalentTo(unexpected, config);
```

Comment:
What about adding methods called `SubjectShouldXXXX(...)` which just calls `XXX(...)`? It's just for the fluent readability 😉

Reply:
Or... 😉 
```cs
Subject.Should().Subject.Should().Subject.Should().Subject.Should().Foo();
```

Reply:
Ok.. got it ;)

Reply:
Tounge-Twister ;)